src/managed/CommonManaged.props

```diff
@@ -17,6 +34,21 @@
     <PackageThirdPartyNoticesFile>$(RepoRoot)THIRD-PARTY-NOTICES.TXT</PackageThirdPartyNoticesFile>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(StabilizePackageVersion)' != 'true'">
+    <!-- Dev builds get a minor version, by default, of '0'.  Our tests restore packages with
```

Comment:
You are right - we aren't using `*` versions anymore. Looks like @mellinoe fixed it.

Here's what we are doing now:
https://github.com/dotnet/core-setup/search?utf8=%E2%9C%93&q=MNAVersion&type=

I'll remove this, and remove it from the dir.props I copied it from as well:
https://github.com/dotnet/core-setup/blob/674c3e3f09284c32321bd9eabd60852e02c0aefb/dir.props#L71-L75